recipes/cad1/task1/baseline/enhance.py

```diff
@@ -142,10 +143,12 @@ def map_to_dict(sources: np.ndarray, sources_list: list[str]) -> dict:
 
 # pylint: disable=unused-argument
 def decompose_signal(
```

Comment:
I'd suggest moving the model_name logic to the calling function. Having both model and model_name passed to this lower level function invite misuse, i.e. passing a demucs model but setting name to be something else.  This function is basically: do some optional normalization, do some optional resampling and then separate the signals. It shouldn't have the model name as a parameter.  The enhance() function should be using this name to load the correct model and then to set up the call to this function.

Reply:
Yeah, I was unsure about how to deal with the model name. 
I may need to move the resampling and normalization to enhance()